apps/examples/araplayer/AudioQueue.cpp

```diff
@@ -0,0 +1,44 @@
+/* ****************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+#include "AudioQueue.h"
+
+AudioQueue::AudioQueue()
+{
+}
+AudioQueue::~AudioQueue()
+{
+}
+
+std::function<void()> AudioQueue::deQueue()
```

Comment:
In future, let's move both of media f/w and this to use common Queue after create common queue in  framework/common/commandQueue